src/variations.js

```diff
@@ -0,0 +1,320 @@
+
+      // Disable TP UI.
+      "browser.contentblocking.trackingprotection.ui.enabled": false,
+      "browser.contentblocking.trackingprotection.control-center.ui.enabled": false,
+    },
+  },
+
+  "FB5L1": {
+    weight: 1,
+    prefs: {
+      // Always turn on Content Blocking.
+      "browser.contentblocking.enabled": true,
+      "browser.contentblocking.ui.enabled": true,
+
+      // Make sure we're not affected by the Symantec distrust.
+      "security.pki.distrust_ca_policy": 1,
+
+      "browser.fastblock.enabled": true,
+      "browser.fastblock.timeout": 5000,
+      "privacy.trackingprotection.enabled": false,
+      "network.http.tailing.enabled": false,
+
+      "urlclassifier.trackingAnnotationTable": "test-track-simple,fastblock1-track-digest256",
+      "urlclassifier.trackingAnnotationWhitelistTable": "test-trackwhite-simple,mozstd-trackwhite-digest256,fastblock1-trackwhite-digest256",
+
+      // Disable TP UI.
+      "browser.contentblocking.trackingprotection.ui.enabled": false,
+      "browser.contentblocking.trackingprotection.control-center.ui.enabled": false,
+    },
+  },
+
+  "FB5L2": {
+    weight: 1,
+    prefs: {
+      // Always turn on Content Blocking.
+      "browser.contentblocking.enabled": true,
+      "browser.contentblocking.ui.enabled": true,
+
+      // Make sure we're not affected by the Symantec distrust.
+      "security.pki.distrust_ca_policy": 1,
+
+      "browser.fastblock.enabled": true,
+      "browser.fastblock.timeout": 5000,
+      "privacy.trackingprotection.enabled": false,
+      "network.http.tailing.enabled": false,
+
+      "urlclassifier.trackingAnnotationTable": "test-track-simple,fastblock2-track-digest256",
+      "urlclassifier.trackingAnnotationWhitelistTable": "test-trackwhite-simple,mozstd-trackwhite-digest256,fastblock2-trackwhite-digest256",
+
+      // Disable TP UI.
+      "browser.contentblocking.trackingprotection.ui.enabled": false,
+      "browser.contentblocking.trackingprotection.control-center.ui.enabled": false,
+    },
+  },
+
+  "FB5L3": {
+    weight: 1,
+    prefs: {
+      // Always turn on Content Blocking.
+      "browser.contentblocking.enabled": true,
+      "browser.contentblocking.ui.enabled": true,
+
+      // Make sure we're not affected by the Symantec distrust.
+      "security.pki.distrust_ca_policy": 1,
+
+      "browser.fastblock.enabled": true,
+      "browser.fastblock.timeout": 5000,
+      "privacy.trackingprotection.enabled": false,
+      "network.http.tailing.enabled": false,
+
+      "urlclassifier.trackingAnnotationTable": "test-track-simple,fastblock3-track-digest256",
+      "urlclassifier.trackingAnnotationWhitelistTable": "test-trackwhite-simple,mozstd-trackwhite-digest256,fastblock3-trackwhite-digest256",
+
+      // Disable TP UI.
+      "browser.contentblocking.trackingprotection.ui.enabled": false,
+      "browser.contentblocking.trackingprotection.control-center.ui.enabled": false,
+    },
+  },
+
+  "Control": {
```

Comment:
should be 4 control branches each with a weight of 1.
like control0, control1... etc